src/apple/templates.ts

```diff
@@ -0,0 +1,47 @@
+export function getRunScriptTemplate(
+    orgSlug: string,
+    projectSlug: string,
+    apiKey: string,
+    uploadSource = true,
+): string {
+    // eslint-disable-next-line no-useless-escape
+    return `# This script is responsable to upload debug symbols and source context for Sentry.\\nif which sentry-cli >/dev/null; then\\nexport SENTRY_ORG=${orgSlug}\\nexport SENTRY_PROJECT=${projectSlug}\\nexport SENTRY_AUTH_TOKEN=${apiKey}\\nERROR=$(sentry-cli debug-files upload ${uploadSource ? "--include-sources " : ""}\"$DWARF_DSYM_FOLDER_PATH\" 2>&1 >/dev/null)\\nif [ ! $? -eq 0 ]; then\\necho \"warning: sentry-cli - $ERROR\"\\nfi\\nelse\\necho \"warning: sentry-cli not installed, download from https://github.com/getsentry/sentry-cli/releases\"\\nfi\\n`;
+}
+
+export const scriptInputPath = "\"${DWARF_DSYM_FOLDER_PATH}/${DWARF_DSYM_FILE_NAME}/Contents/Resources/DWARF/${TARGET_NAME}\"";
+
+export function getSwiftSnippet(dsn: string): string {
+    return `           SentrySDK.start { options in
+            options.dsn = "${dsn}"
+            options.enableTracing = true
+            #if DEBUG
+            options.debug = true
+            options.environment = "Development"
+            #else
+            options.environment = "Release"
+            #endif
+            options.attachScreenshot = true
+            options.attachViewHierarchy = true
+            options.enableTimeToFullDisplay = true
```

Comment:
`m`: I'm unsure if we should enable that by default. If you don't call the manual API the transactions will look a bit weird.

Reply:
what else would be required to give the best product experience? Does it require manual instrumentation which would be way too complicated for the wizard to manage at this stage?

Reply:
I think we're good with the automatic instrumentations